requirements.txt

```diff
@@ -15,25 +15,26 @@ redis==3.3.11                   # Broker for celery
 PyMuPDF==1.16.5
 
 # third party deps
-django-filebrowser==3.11.1      # File uploading
+# django-filebrowser==3.12.1      # File uploading
+git+https://github.com/sehmaschine/django-filebrowser.git@4c1a86d67cd7424df57b0e6880152b309823f009#egg=django-filebrowser==3.12.2
```

Comment:
I've previously been advised to do it this way because otherwise it could break when the actual `3.13.0` is released, but I can probably do it :man_shrugging:

Reply:
But wouldn't it break anyway when a potential `3.12.2` is released, then?

Reply:
I don't think that would be a problem, because we wouldn't install 3.12.2, only later versions.
When that is said I can't remember exactly what the advice stated, so we can take a chance and just name it correctly?

Reply:
It would say take a chance 🎲

Reply:
:ok: